tests/bcl-test/templates/common/TestRunner.Core/TcpTextWriter.cs

```diff
@@ -62,25 +64,41 @@ static string SelectHostName (string[] names, int port)
 			return result;
 		}
 
-		public TcpTextWriter (string hostName, int port)
+		public TcpTextWriter (string hostName, int port, bool isTunnel = true)
```

Comment:
Tunnel defaults to `true` here - but it default to `false` somewhere else (confusing to use)